packages/medusa-payment-stripe/src/services/stripe-provider.js

```diff
@@ -236,37 +229,26 @@ class StripeProviderService extends PaymentService {
    * @returns {object} Stripe payment intent
    */
   async updatePayment(sessionData, cart) {
-    try {
-      const stripeId = cart.customer?.metadata?.stripe_id || undefined
-
-      if (stripeId !== sessionData.customer) {
-        return this.createPayment(cart)
-      } else {
-        if (cart.total && sessionData.amount === Math.round(cart.total)) {
-          return sessionData
-        }
-
-        return this.stripe_.paymentIntents.update(sessionData.id, {
-          amount: Math.round(cart.total),
-        })
+    const stripeId = cart.customer?.metadata?.stripe_id || undefined
+    if (stripeId !== sessionData.customer) {
+      return this.createPayment(cart)
+    } else {
+      if (cart.total && sessionData.amount === Math.round(cart.total)) {
+        return sessionData
       }
-    } catch (error) {
-      throw error
+
+      return this.stripe_.paymentIntents.update(sessionData.id, {
+        amount: Math.round(cart.total),
+      })
     }
   }
 
   async deletePayment(payment) {
-    try {
-      const { id } = payment.data
-      return this.stripe_.paymentIntents.cancel(id).catch((err) => {
-        if (err.statusCode === 400) {
-          return
-        }
-        throw err
-      })
-    } catch (error) {
-      throw error
-    }
+    const { id } = payment.data
+    return this.stripe_.paymentIntents.cancel(id).catch((err) => {
+      if (err.statusCode === 400) return
```

Comment:
we prefer to always have brackets 
```suggestion
      if (err.statusCode === 400) {
        return
      }
```